README.md

```diff
@@ -292,6 +302,58 @@ css({
 
 ### Do: make use of multiple classes
 
+```js
+const MyComponent = () => {
+  return (
+    <ul className={className}>
+      {navigationLinks.map((nav, idx) => (
+        <li
+          key={idx}
+          className={`nav-item ${nav.href === window.location.pathname ? 'active' : ''`}
+        >
+          <a href={x.href}>{x.linkText}</a>
+        </li>
+      )}
+    </ul>
+  )
+}
+
+const className = css({
+  '& .nav-item': { cursor: 'pointer' },
+  '& .nav-item.active': { cursor: 'not-allowed' },
+})
+```
+
+### Do: make use of helper functions
```

Comment:
Is this example just to show you can use the `classnames` package with this `cssinjs` package?

Reply:
Yes, exactly. Any helper function really, but this one is mentioned in the React docs.